src/tests/stringy.spec.ts

```diff
@@ -53,4 +59,63 @@ describe('Stringy methods', () => {
       expect(result).to.equal(true);
     });
   });
+
+  describe('isSuffix', () => {
+    const cashew = new Stringy('cashew');
+    const cashewWithStartingSpace = new Stringy(' cashew');
+    const casHEW = new Stringy('HEW');
```

Comment:
Codacy has a fix for the issue: [Replace `'HEW'` with `"HEW"`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
    const casHEW = new Stringy("HEW");
```